internal/cache/file/cache_handle.go

```diff
@@ -32,26 +32,27 @@ type CacheHandle struct {
 	// fileHandle to a local file which contains locally downloaded data.
 	fileHandle *os.File
 
-	//	fileDownloadJob is a reference to async download Job.
+	// fileDownloadJob is a reference to async download Job.
```

Comment:
Can we write a comment about - it's life-cycle. Like,  this will become nil once Job is finished/cancelled or invalidated.